conf/locale/eo/LC_MESSAGES/django.po

```diff
@@ -37,8 +37,8 @@ msgid ""
 msgstr ""
```

Comment:
Note to reviewers: Apologies for including this in the PR, but I needed some dummy translation strings to verify that the language override was working correctly.